geode-core/src/main/java/org/apache/geode/internal/cache/tier/sockets/CacheClientProxyStats.java

```diff
@@ -128,7 +133,10 @@
             "operations"),
 
         f.createLongCounter(CQ_COUNT, "Number of CQs on the client.", "operations"),
-        f.createLongCounter("sentBytes", "Total number of bytes sent to client.", "bytes"),});
+        f.createLongCounter("sentBytes", "Total number of bytes sent to client.", "bytes"),
+        f.createIntCounter(NUMBER_THREADS_PUT_IN_QUEUE,
```

Comment:
This should be a gauge not a counter. Counters always increase. Also all the "int" ones are deprecated so just make it "createLongGauge"